components/backup-stream/src/endpoint.rs

```diff
@@ -619,7 +619,6 @@ where
             let task_clone = task.clone();
             let run = async move {
                 let task_name = task.info.get_name();
-                cli.init_task(&task.info).await?;
```

Comment:
why remove this?

Reply:
This is a legacy code which initializes some keys not being used any more.

Reply:
(Also this relies on the `Txn` API which isn't supported by the PD API.)